core-java-modules/core-java-string-conversions-2/src/test/java/com/baeldung/stringtocamelcase/StringtocamelcaseApplicationTests.java

```diff
@@ -0,0 +1,61 @@
+package com.baeldung.stringtocamelcase;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class StringtocamelcaseApplicationTests {
+
+	private static final String TEXT_WHITESPACE = "THIS STRING SHOULD BE IN CAMEL CASE";
+	private static final String TEXT_UNDERSCORE = "THIS_STRING_SHOULD_BE_IN_CAMEL_CASE";
+	private static final String TEXT_EXPECTED = "ThisStringShouldBeInCamelCase";
+
+	@Test
+	public void testNothing() {
+	}
+
+	@Test
+	public void testToCamelCaseByIteration_shouldReturnCamelCase() {
```

Comment:
test names should not start `test` (all tests are tests and have @Test in front already so `test` adds nothing). Try to use the `given`/`when`/`then` naming.